roles/download/tasks/download_container.yml

```diff
@@ -59,7 +59,7 @@
       retries: 4
       become: "{{ user_can_become_root | default(false) or not download_localhost }}"
       when:
-        - pull_required
```

Comment:
hmm this is from my PR fix. need rebase ?

Reply:
Strange indeed I'll rebase